torch/_dynamo/convert_frame.py

```diff
@@ -529,8 +528,10 @@ def compile_inner(
         transform: Callable[[List[Instruction], Dict[str, Any]], Any],
     ) -> Optional[GuardedCode]:
         nonlocal output
+        nonlocal wasted_compile_time
```

Comment:
TODO: "wasted" might not be the right word to describe the time spent on an attempt that triggered a restart. Maybe "extra_compile_time"?

Reply:
I think ```restart_count``` would be a better metrics, as one restart take 100 sec is better than 10 restarts takes 100 sec. We should figure out why it restarts so many times. If only one restart but take too long time, this means the dynamo tracing of this frame is pretty slow, we can get the signal from ```entire_frame_compile_time - backend_compile_time```.

Reply:
I don't like "wasted" either. Maybe `dynamo_time_before_restart`